f2/src/locales/en.json

```diff
@@ -222,7 +222,7 @@
   "scammerDetail.fileDescription": "Describe what this file shows",
   "scammerDetail.intro": "Any clues about the source of the scam can help police track down the scammer.",
   "scammerDetail.nextButton": "Next: Impact of scam",
-  "scammerDetail.nextPage": "scammerDetail.nextPage",
+  "scammerDetail.nextPage": " ",
```

Comment:
This is old p2 code. Just changed this to pass the tests, This line will be deleted when we clean up the repo.